CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#5561](https://github.com/thanos-io/thanos/pull/5561) Query Frontend: Support instant query vertical sharding.
 - [#5453](https://github.com/thanos-io/thanos/pull/5453) Compact: Skip erroneous empty non `*AggrChunk` chunks during 1h downsampling of 5m resolution blocks.
 - [#5607](https://github.com/thanos-io/thanos/pull/5607) Query: Support custom lookback delta from request in query api.
+- [#5561](https://github.com/thanos-io/thanos/pull/5650) Query Frontend: Add sharded queries metrics.
```

Comment:
Please add this change to the `Added` section in `Unreleased`.

Reply:
right - probably messed it up during a merge, fixed it now.